config/routes.rb

```diff
@@ -7,7 +7,7 @@
 
   delete '/logout', to: 'sessions#destroy', as: 'logout'
 
-  get '/cart', to: "sessions#showcart", as: "showcart"
+  get '/cart', to: "carts#showcart", as: "showcart"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.